Tests/js/views/services/plugins/assets/ez-userloadplugin-tests.js

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-userloadplugin-tests', function (Y) {
+    var tests, registerTest,
+        Assert = Y.Assert;
+
+    tests = new Y.Test.Case({
+        name: "eZ User Load Plugin event tests",
+
+        setUp: function () {
+            this.User = function () {
+                this.set = tests._set;
+                this.load = tests._load;
+            };
+
+            this.userId = '/my/user/42';
+            this.capi = {};
+            this.service = new Y.Base();
+            this.view = new Y.View();
+            this.view.set('myUser', null);
+            this.view.set('loadingError', false);
+            this.view.addTarget(this.service);
+            this.service.set('capi', this.capi);
+
+            this.plugin = new Y.eZ.Plugin.UserLoad({
+                host: this.service,
+            });
+
+            this.plugin.set('userModelConstructor', this.User);
+        },
+
+        _set: function (name, value) {
+            var isCorrectValue = false;
+            Assert.areSame(
+                'id',
+                name,
+                'method set should be called to set the id'
+            );
+            if ( value == tests.userId ){
+                isCorrectValue = true;
+            }
+            Assert.isTrue(isCorrectValue, 'The id should be the user id');
+            this.id = value;
```

Comment:
it is called by the `_loadUser` function `loadedUser.set('id', e.userId);` 

And then we check that id has been set properly:
```
Assert.areSame(
    this.userId,
    this.view.get('myUser').id,
    "The view should get the user"
 );
```